Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+# v1.8.1
+- [fixed] Firestore no longer loads its TLS certificates from a bundle, which
+  fixes crashes at startup when the bundle can't be loaded. This fixes a
+  specific case where the bundle couldn't be loaded due to international
+  characters in the application name. If you're manually tracking dependencies,
+  you can now remove `gRPCCertificates-Cpp.bundle` from your build. (#3951).
```

Comment:
>If you're manually tracking dependencies, you can now remove `gRPCCertificates-Cpp.bundle` from your build.

This is true for Carthage users as well, right?

Reply:
That's right.